platform/platform.go

```diff
@@ -0,0 +1,18 @@
+package platform
+
+//Platform system platform
+var Platform string
+
+const (
+	//defaultPlatform platform x86_64
+	defaultPlatform = "x86_64"
```

Comment:
You can use [mainPlatform](https://github.com/openfaas/faas-cli/blob/2487d1d3443c9b7201c98a2ecc57b617ac58db56/commands/template_store_list.go#L24) from the template store it is directly in the commands package so I believe you should be able to access it directly

Reply:
We using this package to basically inject the value at the build time and use the same for that binary. If we use `mainPlatform` variable it will change the behavior of templates commands as well.

Reply:
If that is something we want .. than We can use the same variable.

Reply:
It would be good to consolidate the constants especially since they are used in the same context with the function store.Perhaps update `mainPlatform` to `defaultPlatform` and use that one constant everywhere.

This will make maintenance much easier in the future.

Reply:
What did we decide on?

Reply:
We have removed `platform` package and using an variable in `commands` package name `Platform` which is injected at the build time. if that value is empty then we are using `mainPlatform` as default value for the platform.